### DISCUSSION TITLE
Multi-region cluster

### DISCUSSION THREAD

**Discussion by caniko** [mysticaltech/terraform-hcloud-kube-hetzner#1086] [Q&A]
Would it be possible to create a multi-region (GER, FIN, USA) cluster across data centers? Where nodes are on different regions? Perhaps the control plane could be on the region where the costs are the lowest.

There is no mention of this anywhere in the README, and it is something that I am looking for.

**Comment by maggie44** [comment]
https://github.com/kube-hetzner/terraform-hcloud-kube-hetzner/discussions/883

**Reply by caniko** [reply]
Thanks! Note that this is not multi-cluster, but multi-region cluster. Nodes have far-reaching geolocations in **one** cluster. The question is if Hetzner supports this. The private network needs to be open between Hetzner DCs :vulcan_salute:
